src/initialization/InitDistribution.H

```diff
@@ -0,0 +1,96 @@
+/* Copyright 2022-2023 The Regents of the University of California, through Lawrence
+ *           Berkeley National Laboratory (subject to receipt of any required
+ *           approvals from the U.S. Dept. of Energy). All rights reserved.
+ *
+ * This file is part of ImpactX.
+ *
+ * Authors: Axel Huebl, Andrew Myers
+ * License: BSD-3-Clause-LBNL
+ */
+#ifndef IMPACTX_INITIALIZATION_INITDISTRIBUTION_H
+#define IMPACTX_INITIALIZATION_INITDISTRIBUTION_H
+
+#include "particles/ImpactXParticleContainer.H"
+
+#include <AMReX_Extension.H> // for AMREX_RESTRICT
+#include <AMReX_REAL.H>
+
+#include <type_traits>
```

Comment:
For `std::move`:
```suggestion
#include <utility>

```
`type_traits` is not used here.